src/views/pages/ProductTransactions/index.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class='ProductTransactions'>
+    <md-table v-model="productTransactions" md-sort="name" md-sort-order="asc" md-card>
+      <md-table-toolbar>
+        <div class="md-toolbar-section-start">
+          <h1 class="md-title">Product Transactions</h1>
+        </div>
+        <md-button v-on:click="downloadCsv" class="md-raised md-primary download_csv_button">Download CSV</md-button>
+      </md-table-toolbar>
+
+      <h2 class="error md-subheader">{{error}}</h2>
+
+      <md-table-row slot="md-table-row" slot-scope="{ item }">
+        <md-table-cell md-label="Type" md-sort-by="productType">{{item.productType}}</md-table-cell>
+        <md-table-cell md-label="Date Time" md-sort-by="datetime">{{item.datetime}}</md-table-cell>
+        <md-table-cell md-label="Amount" md-sort-by="amountOfProduct">{{item.amountOfProduct}}</md-table-cell>
+        <md-table-cell md-label="Units" md-sort-by="productUnits">{{item.productUnits}}</md-table-cell>
+        <md-table-cell md-label="Cost per Unit" md-sort-by="costPerUnit">{{item.costPerUnit}}</md-table-cell>
+        <md-table-cell md-label="Currency" md-sort-by="currency">{{item.currency}}</md-table-cell>
+        <md-table-cell md-label="Last Modified" md-sort-by="lastModified">{{item.lastModified}}</md-table-cell>
+      </md-table-row>
+    </md-table>
+  </div>
+</template>
+
+<script lang='ts'>
+import Vue from 'vue';
+import axios from 'axios';
+
+interface ProductTransaction {
+  productType: string;
+  datetime: string;
+  amountOfProduct: number;
+  productUnits: string;
+  costPerUnit: number;
+  currency: string;
+  lastModified: string;
+}
+
+export default Vue.extend({
+  name: 'ProductTransactions',
+  methods: {
+    downloadCsv: function downloadCsv() {
+      alert('TODO bea :)');
+    },
+  },
+  created: async function created() {
```

Comment:
hmm this is confusing? this doesn't actually create anything?

Reply:
This is a built in special function for vue.